pkg/process/process_test.go

```diff
@@ -0,0 +1,96 @@
+package process
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+)
+
+func TestSendSignal(t *testing.T) {
+	//create a new process
+	p := exec.Command("sleep", "10")
+
+	//start the process
+	if err := p.Start(); err != nil {
+		t.Error("unable to start process: %w", err)
+	}
+
+	if err := SendSignal(p.Process.Pid, os.Kill); err != nil {
+		t.Error("unable to send signal to process: %w", err)
+	}
+
+	//release the pid
+	p.Wait()
+
+	//check if process exists
+	exists, err := Exists(p.Process.Pid)
+	if err != nil {
+		t.Error("error while retrieving process: %w", err)
+	}
```

Comment:
gomega matchers here (and everywhere) please:
```suggestion
	g.Expect(err).NotTo(HaveOccurred())
```